data/SkyPy/euclid-roman-lsst-like.yml

```diff
@@ -1,6 +1,6 @@
 mag_lim: 30
 fsky: 0.1 deg2
-z_range: !numpy.arange [0, 5.01, 0.1]
+z_range: !numpy.arange [0, 3.01, 0.1]
```

Comment:
remind me, z>3 is not possible because of how the filters are defined in Euclid speclite that go too blue. Perhaps make a comment here